src/js/bootstrap-slider.js

```diff
@@ -898,10 +898,14 @@ const windowIsDefined = (typeof window === "object");
 				}
 			},
 
-			setValue: function(val, triggerSlideEvent, triggerChangeEvent) {
+			setValue: function(val, triggerSlideEvent, triggerChangeEvent, originalEvt) {
```

Comment:
could you update the README to reflect this new change to the method signature?

https://github.com/seiyria/bootstrap-slider#functions